### ISSUE TITLE
std_preclust() at line 270

### ISSUE THREAD

**Issue by evastone97** [eeglabdevelopers/eeglab#471]
Hello again, 

This is the [study ](https://drive.google.com/drive/folders/1wuYYs8BrPcpVnPuzc9ubHrdpnhCuFuKT?usp=sharing)that I am using and I am getting this specific error when trying to precompute measures. 

![image](https://user-images.githubusercontent.com/82998648/167715002-b6b6cb01-7d90-4bdf-a940-1fc12e7071b3.png)

What might be the reason for that, can I change something in the function? (I have already download the functions with the changes that you have made but I still have the error )

**Comment by arnodelorme** [comment]
Can you share the whole STUDY (including the data)?

**Comment by evastone97** [comment]
ok with all the files and precalculations?

**Comment by evastone97** [comment]
I will send the study link with all the subfiles (they are a lot of files so I am trying to find the most optimal way to send them)

So 18subjects with 3 different conditions have been added to a study and I have done ICA, and precompute component measures.  When I press the button to build the clustering array I got this warning:

![image](https://user-images.githubusercontent.com/82998648/167952782-b53ccc97-f8a5-4fb9-a7f9-65114debb936.png)

Afterward, when I press ok and try to plot the ERPs, scalp maps, and spectra I got the warning for std_preclust in line 270 which makes sense because I haven't just chosen the scalp maps. 

![image](https://user-images.githubusercontent.com/82998648/167953272-0d947e02-23b2-480b-a2ea-cd181ca16d94.png)

You mentioned in another issue that I should run again ICA but it didn't work. 

Thank you in advance!
